src/main/java/org/mule/extension/file/internal/FileOperations.java

```diff
@@ -156,12 +160,16 @@
   public void write(@Config FileConnectorConfig config, @Connection FileSystem fileSystem,
                     @Path(type = FILE, location = EXTERNAL) String path,
                     @Content @Summary("Content to be written into the file") InputStream content,
-                    @Optional @Summary("Encoding when trying to write a String file. If not set, defaults to the configuration one or the Mule default") @Placement(
-                        tab = ADVANCED_TAB) String encoding,
+                    @Optional @Summary("This parameter is deprecated and will not be taken into account") @Placement(
```

Comment:
"Deprecated: This parameter will not be taken into account for the operation execution"